modules/ztp-provisioning-edge-sites-at-scale.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="provisioning-edge-sites-at-scale_{context}"]
+= Provisioning edge sites at scale
+
+Telco edge computing presents extraordinary challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands of locations. These challenges require fully-automated management solutions with, as closely as possible, zero human interaction.
+
+Zero touch provisioning (ZTP) allows you to provision new edge sites without manual configuration of bare-metal equipment at remote sites. Instead, you push declarative frameworks of desired end states out into the environment. Template or overlay configurations install {product-title} features that are required for CNF workloads. End-to-end functional test suites are used to verify CNF related features. All Kustomize configs are entirely declarative in nature.
+
+You start by creating and refining ISO images of the node configuration you want. Create a unique image for each node based on the configuration. The images are then used to repeatedly provision large numbers of nodes efficiently and quickly, allowing you keep up with the requirements from the field for far edge nodes.
```

Comment:
this is misleading - the workflow based on the declarative configuration creates the iso's the ISO's can then be delivered to the edge nodes to begin the installation process.  Delete/rework the first two sentences

Reply:
reworded as: "Zero touch provisioning (ZTP) allows you to provision new edge sites with declarative configurations of bare-metal equipment at remote sites. Template or overlay configurations install {product-title} features that are required for CNF workloads. End-to-end functional test suites are used to verify CNF related features. All configurations are declarative in nature."